apps/wallet-mobile/src/features/Receive/useCases/ListMultipleAddressesScreen.tsx

```diff
@@ -43,9 +47,26 @@ export const ListMultipleAddressesScreen = () => {
   const {openModal} = useModal()
   const {isShowingMultipleAddressInfo} = useMultipleAddressesInfo()
 
+  const {next: nextReceiveSingleAddress} = useReceiveAddressesStatus('single')
+
+  const handleOnModalConfirm = React.useCallback(
+    (method: AddressMode) => {
+      if (method === 'single') {
+        selectedAddressChanged(nextReceiveSingleAddress)
+        navigate.replaceReceiveSingle()
+      }
+    },
+    [navigate, nextReceiveSingleAddress, selectedAddressChanged],
+  )
+
   React.useEffect(() => {
-    isShowingMultipleAddressInfo && openModal(strings.multiplePresentation, <Modal />, modalHeight)
-  }, [isShowingMultipleAddressInfo, openModal, strings.multiplePresentation])
+    isShowingMultipleAddressInfo &&
+      openModal(
```

Comment:
I think the idea is to never show it in this screen, only on the DescribeSelectedAddressScreen. 
If we are on this screen, it means we are in Multi mode, and only way to be in Multi mode should be for the user to manually have set it.